tests/api_fastapi/core_api/routes/public/test_dag_sources.py

```diff
@@ -0,0 +1,103 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import ast
+import os
+
+import pytest
+from httpx import Response
+
+from airflow.models.dag import DAG
+from airflow.models.dagbag import DagBag
+
+from tests_common.test_utils.db import clear_db_dag_code, clear_db_dags, clear_db_serialized_dags
+
+pytestmark = pytest.mark.db_test
+
+ROOT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
```

Comment:
All of those hardcoded path do not look right. There is most likely constants already defined in the test folder or the global airflow config to not have to manually do all this.

Reply:
Done, have removed `ROOT_DIR`, thanks! 

JFTR, I got that here when I migrated the constants from here: https://github.com/apache/airflow/blob/main/tests/api_connexion/endpoints/test_dag_source_endpoint.py#L35